src/sentry-global-search/lib/types.ts

```diff
@@ -44,6 +44,13 @@ export type SearchHit = ObjectWithObjectID & {
    */
   _highlightResult?: { [k: string]: { value: string } };
   _snippetResult?: { [k: string]: { value: string } };
+  /**
+   * String representing secondary context information.
+   */
+  category?: {
+    id?: string,
```

Comment:
I would just add a comment with an example category here, it makes it easy for whoever is reading this to understand what it represents.

Reply:
fixed in edcd2d58ebde045da2e32e9d90088b2f9e802432